plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -55,36 +55,39 @@
   appname:
     type: str
     description:
-      - Name of the application
+      - Name of the application.
+      - This option has been deprecated and will be removed. Please do not use.
```

Comment:
```suggestion
      - This option has been deprecated and will be removed in community.general 7.0.0. Please do not use.
```
(Same for the other two deprecated options.)